metis/spec/folder_spec.rb

```diff
@@ -19,6 +19,38 @@ def app
     expect(stubs.contents(:athena)).to be_empty
   end
 
+  context '#size' do
+    before(:each) do
+      @wisdom_file = create_file('athena', 'wisdom.txt', WISDOM)
+      stubs.create_file('athena', 'files', 'wisdom.txt', WISDOM)
+
+      @blueprints_folder = create_folder('athena', 'blueprints')
+      stubs.create_folder('athena', 'files', 'blueprints')
+
+      @helmet_folder = create_folder('athena', 'helmet', folder: @blueprints_folder)
+      stubs.create_folder('athena', 'files', 'blueprints/helmet')
+
+      @second_helmet_folder = create_folder('athena', 'helmet')
+      stubs.create_folder('athena', 'files', 'helmet')
+
+      @helmet_file = create_file('athena', 'helmet.jpg', HELMET, folder: @helmet_folder)
+      stubs.create_file('athena', 'files', 'blueprints/helmet/helmet.jpg', HELMET)
+    end
+    it 'returns total size of a bucket' do
+      token_header(:editor)
+      get('/athena/size/files/')
+      expect(last_response.status).to eq(200)
+      expect(json_body[:size]).to eq(79)
```

Comment:
Is there a way you don't have to manually calculate? Maybe like `(WISDOM + HELMET).bytesize`